views/pages/RegisterPage.jsx

```diff
@@ -1,126 +1,48 @@
-  const handleFormSubmit = async (data) => {
+  const onSubmit = async (data) => {
     try {
-      if (data.password !== data.confirmPassword) {
-        setErrors({
-          ...errors,
-          password: { error: true, message: 'passwords do not match' },
-        })
-
-        return
-      } else {
-        setErrors({
-          ...errors,
-          password: { error: false, message: '' },
-        })
-      }
-
-      const isFormReadyToBeSubmitted = Object.values(errors).every(
-        (value) => value.error === false
-      )
-
-      if (isFormReadyToBeSubmitted) {
-        await api.auth.signup(data)
-        setNotification({
-          open: true,
-          message:
-            'Account has been created, please wait for an Admin to provide access.',
-        })
-      } else {
-        setNotification({
-          open: true,
-          message: 'There are still errors in your form.',
-        })
-      }
+      await api.auth.signup(data)
+      setNotification({
+        open: true,
+        message:
+          'Account has been created, please wait for an Admin to provide access.',
+      })
     } catch (error) {
       setNotification({
         open: true,
         message: error.message,
       })
     }
   }
-  const handleChange = (e) => {
-    const { name, value } = e.target
-
-    if (name === 'email') {
-      if (VALIDATION_EMAIL_REGEX.test(value)) {
-        setErrors({
-          ...errors,
-          email: { error: false, message: '' },
-        })
-      } else if (!VALIDATION_EMAIL_REGEX.test(value)) {
-        setErrors({
-          ...errors,
-          email: { error: true, message: 'incorrect email format' },
-        })
-      }
-    }
-    if (name === 'password' || name === 'confirmPassword') {
-      if (value.length < 8) {
-        setErrors({
-          ...errors,
-          [name]: {
-            error: true,
-            message: 'passwords must be a minimum of 8 characters',
-          },
-        })
-      } else {
-        setErrors({
-          ...errors,
-          [name]: { error: false, message: '' },
-        })
-      }
-    }
-  }
   return (
-    <div>
-      <Card>
-        <div>
-          <CardContent>
-            <Typography variant="title">Welcome to DPdash!</Typography>
-            <Typography variant="subheading" color="textSecondary">
-              Please create your DPdash account to continue.
-            </Typography>
-          </CardContent>
-          <div>
-            <div>
-              <RegistrationForm
-                control={control}
-                onSubmit={handleSubmit(handleFormSubmit)}
-                errors={errors}
-                navigate={navigate}
-                onInputChange={handleChange}
-              />
-            </div>
-          </div>
-        </div>
-      </Card>
-    </div>
+    <>
+      <Typography variant="h4">Welcome to DPdash!</Typography>
+      <Typography variant="body1" color="textSecondary">
```

Comment:
```suggestion
      <Typography variant="body1" color="text.secondary">
```
I think I made this typo